src/app/qtlocalpeer/qtlocalpeer.cpp

```diff
@@ -114,8 +115,10 @@ QtLocalPeer::QtLocalPeer(QObject *parent, const QString &appId)
 
     QByteArray idc = id.toUtf8();
     quint16 idNum = qChecksum(idc.constData(), idc.size());
-    socketName = QLatin1String("qtsingleapp-") + prefix
-                 + QLatin1Char('-') + QString::number(idNum, 16);
+    socketName = Profile::instance()->location(SpecialFolder::Config) + QLatin1Char('/')
```

Comment:
Seems we can omit all such generating of file name uniqueness since the path is unique now. Am I wrong?

Reply:
you are right, removed now.